Gems/Atom/RHI/Vulkan/Code/Source/RHI/VkAllocator.cpp

```diff
@@ -0,0 +1,61 @@
+/*

```

Comment:
This file needs to also be relocated to RHI.Reflect folder.

Reply:
done